lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -32,17 +32,23 @@ const UIStrings = {
     'children/parent element. A large DOM can increase memory usage, cause longer ' +
     '[style calculations](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations), ' +
     'and produce costly [layout reflows](https://developers.google.com/speed/articles/reflow). [Learn more](https://developers.google.com/web/tools/lighthouse/audits/dom-size).',
-  /** Label for the total number of DOM nodes found in the page. */
-  columnDOMNodes: 'Total DOM Nodes',
-  /** Label for the numeric value of the maximum depth in the page's DOM tree. */
-  columnDOMDepth: 'Maximum DOM Depth',
-  /** Label for the value of the maximum number of children any DOM node in the page has. */
-  columnDOMWidth: 'Maximum Children',
+  /** Table column header for the type of analytic found. */
+  columnCategory: 'Category',
```

Comment:
bikeshed. wdyt about 'Statistic' or 'Metric'. cc @brendankenny @patrickhulce

Reply:
I took the Category name from the `Minimizes main thread work` Audit table.  So we can update the phrasing in every table if we like Metric more (which I agree is more descriptive than Category).

![category_screenshot](https://user-images.githubusercontent.com/6392995/45841133-b4a79800-bccd-11e8-8ce1-df63e545f4a1.png)

Reply:
heh, in that case I feel like "Category" is better (since it's actually bringing e.g. multiple Script Evaluation tasks together and grouping them into one line), but also see the appeal of consistency